src/fal/utils.py

```diff
@@ -4,11 +4,13 @@
 from typing import List
 
 
-def print_run_info(models: List[DbtModel], keyword: str, before):
+def print_run_info(models: List[DbtModel], keyword: str, before, scripts=None):
```

Comment:
not passing this new arg in any of the 2 calls you use

Reply:
Good catch! Updated.